### DISCUSSION TITLE
Observação - NFNotaInfo.java

### DISCUSSION THREAD

**Discussion by gabrielyrio** [wmixvideo/nfe#786] [Ideias]
Olá pessoal! Tudo com vocês?

   Uma observação quando à classe NFNotaInfo.java
   Na documentação a tag "pag" Grupo de Formas de Pagamento  é uma lista 
   como ocorrência 0-100.
  No projeto ainda consta com um único objeto.
 
  // Como está no projeto
    @Element(name = "pag")
    private NFNotaInfoPagamento pagamento;
    
  // Como eu uso 
    @ElementList(entry = "pag", inline = true, required = false)
    private List<NFNotaInfoPagamento> pagamento;

Fica aí a sugestão.

Abraços,

Gabriel Cruz Lyrio  27 999411376
gabrielcruzlyrio@gmail.com

**Comment by gabrielyrio** [comment]
Faltou o get e o set

    public List<NFNotaInfoPagamento> getPagamento() {
        return this.pagamento;
    }

    public void setPagamento(final List<NFNotaInfoPagamento> pagamento) {
        this.pagamento = pagamento;
    }

**Comment by marcoslombardi** [comment]
Gabriel,

Obrigado pela observação e sugestão.
Alterei o código e solicitei um PR para o projeto.
